widget.json

```diff
@@ -33,7 +33,7 @@
     ],
     "assets": [
       "vendor/notification.js",
-      "vendor/autosize.min.js",
+      "vendor/autosize.js",
```

Comment:
Was this asset missing?

Reply:
@squallstar no, I just updated to a new version and added the non-minified version because I needed to make some one edit and it was impossible to do it on the minified version!

Reply:
I have added the following condition to their code:
```
// prevents scroll-position jumping on web
// FLIPLET CUSTOM
if (Fliplet.Env.get('platform') === 'web') {    <-- ADDED THIS
  overflows.forEach(function(el) {
    el.node.scrollTop = el.scrollTop;
  });
}
```
I don't want the scroll to be triggered on mobile because the device handles the scrolling automatically, because of the keyboard!

Reply:
Well, basically that was the fix for the issue!